libsel4/arch_include/riscv/interfaces/sel4arch.xml

```diff
@@ -100,12 +100,12 @@
             </description>
             <error name="seL4_IllegalOperation">
                 <description>
-                    The <texttt text="_service"/> is a CPtr to a capability of the wrong type.
+                    The <texttt text="_service"/> is a CPTR to a capability of the wrong type.
```

Comment:
I think these should stay CPtr -- with CPTR I'd expect an acronym. (I might have missed a discussion on this while I was away, apologies if I'm late to to the party)

Reply:
Fair point. I made the (wrong!) choice while trying to fix:

"3: The manual uses CPTR in some places and CPtr in other. We should be consistent. (Assuming the outcome of this issue is updating docs)."
from https://github.com/seL4/seL4/issues/566

I'll change the CPTR to CPtr :-)